lib/vanagon/engine/hardware.rb

```diff
@@ -0,0 +1,72 @@
+require 'vanagon/engine/base'
+require 'json'
+require 'lock_manager'
+
+LOCK_MANAGER_HOST = ENV['LOCK_MANAGER_HOST'] || 'redis'
+LOCK_MANAGER_PORT = ENV['LOCK_MANAGER_PORT'] || 6379
+VANAGON_LOCK_USER = "#{ENV['USER']}"
```

Comment:
You don't need quotes around this or interpolation. ENV entries are always strings or nil.